packages/core/bootstrap/src/lib/ws/epics.ts

```diff
@@ -285,14 +291,21 @@ export const metricsEpic: Epic<AnyAction, AnyAction, any, any> = (action$, state
       const connectionErrorLabels = (payload: WSErrorPayload) => ({
         key: payload.connectionInfo.key,
         url: payload.connectionInfo.url,
-        message: payload.message,
+        message: payload.reason,
       })
       const subscriptionLabels = (payload: WSSubscriptionPayload) => ({
         connection_key: payload.connectionInfo.key,
         connection_url: payload.connectionInfo.url,
         feed_id: getFeedId({ ...payload.input }),
         subscription_key: getSubsId(payload.subscriptionMsg),
       })
+      const subscriptionErrorLabels = (payload: WSSubscriptionErrorPayload) => ({
+        connection_key: payload.connectionInfo.key,
+        connection_url: payload.connectionInfo.url,
+        feed_id: payload.input && getFeedId({ ...payload.input }),
+        message: payload.reason,
+        subscription_key: payload.subscriptionMsg && getSubsId(payload.subscriptionMsg),
```

Comment:
Not sure if this will work correctly in all cases, will it?
Why the event wouldn't send  `payload.input`  or `payload.subscriptionMsg` ?

Reply:
[In this case](https://github.com/smartcontractkit/external-adapters-js/pull/488/files#diff-42dc22535608bc02b8332c4f51fbd00a68974be1dabacdc9f044aaffe01f16f1R129) 
For now we can't identify the subscription from that messags

Reply:
Ok, got it!

But this is going to set it as false, rather do something like this:
```suggestion
        feed_id: payload.input ? getFeedId({ ...payload.input }) : 'N/A',
        message: payload.reason,
        subscription_key: payload.subscriptionMsg ? getSubsId(payload.subscriptionMsg) : 'N/A',
```